### PR TITLE
Bling-Bling

### PR DESCRIPTION
**Pull request by pdyraga**
Added KEEP ASCII to contracts.

### CODE REVIEW

File: contracts/AssetPool.sol [hunk 1]
```diff
@@ -1,3 +1,15 @@
+// ▓▓▌ ▓▓ ▐▓▓ ▓▓▓▓▓▓▓▓▓▓▌▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▄
+// ▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▌▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
+//   ▓▓▓▓▓▓    ▓▓▓▓▓▓▓▀    ▐▓▓▓▓▓▓    ▐▓▓▓▓▓   ▓▓▓▓▓▓     ▓▓▓▓▓   ▐▓▓▓▓▓▌   ▐▓▓▓▓▓▓
+//   ▓▓▓▓▓▓▄▄▓▓▓▓▓▓▓▀      ▐▓▓▓▓▓▓▄▄▄▄         ▓▓▓▓▓▓▄▄▄▄         ▐▓▓▓▓▓▌   ▐▓▓▓▓▓▓
+//   ▓▓▓▓▓▓▓▓▓▓▓▓▓▀        ▐▓▓▓▓▓▓▓▓▓▓         ▓▓▓▓▓▓▓▓▓▓▌        ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
```
Comment (dimpar) [inline contracts/AssetPool.sol:5]: In the first "E" the middle hyphen is different from the second "E". Or we can leave it as is, up to you :)

Comment (pdyraga) [reply contracts/AssetPool.sol:5]: Eagle-eye! We do miss ▌at the beginning and we have an additional ▌at the end, correct?

Comment (dimpar) [reply contracts/AssetPool.sol:5]: that's right

Comment (pdyraga) [reply contracts/AssetPool.sol]: Should be good now @dimpar, please give it another chance.

Comment (dimpar) [reply contracts/AssetPool.sol]: Do we want to add these in the interfaces as well like we have in ex. `keep-ecdsa`?

Comment (pdyraga) [reply contracts/AssetPool.sol]: Of course! Just added.

**Review by dimpar — APPROVED** [review]

**Pull request merged**